draft-ietf-rats-architecture.md

```diff
@@ -697,6 +697,13 @@ for by hardware or by ROM code, especially if such hardware is
 physically resistant to hardware tampering.  The component that is
 implicitly trusted is often referred to as a Root of Trust.
 
+A conveyance protocol that is authenticated and integrity protected can be used
+to convey unprotected Evidence, assuming the following properties exists:
+
+   1. The key used to authenticate and integrity protect the conveyance channel is trusted by the Verifier to speak for the Attesting Environment that collected claims about the Target Environment.
+   2. The Evidence in the conveyance channel is supplied by the Attesting Environment.
```

Comment:
same question about "the Attesting Environment"... which Attesting Environment?  Also the phrase reads as a tautology, can we wordsmith somehow?